src/extensions/yfm/ImgSize/const.ts

```diff
@@ -6,3 +6,13 @@ export {imageNodeName, addImageAction} from '../../markdown/Image/const';
 export const IMG_MAX_HEIGHT = 600; //px
 export type ImageRendererState = {linkAdded: boolean};
 export const imageRendererKey = new PluginKey<ImageRendererState>('imageRenderer');
+
+const knownImageHostsRegexString = '(jing|avatars\\.mds)';
```

Comment:
I suggest putting the regular expression into the parameters of the extension rather than hardcoding it into the source code of the markdown editor.